docs/application/dotnet/api/netstandard-limitations.md

```diff
@@ -1,35 +1,35 @@
 # Limitations of .NET Standard API on Tizen
 
-Since applications on Tizen have limitations compared to .NET desktop applications, it is not possible to use the following facilities:
+Applications on Tizen have limitations compared to .NET desktop applications. Following are the limitations of .NET Standard API on Tizen:
 
 
-1. Reading data from the standard output stream and standard error stream of an application is not allowed.
+1. No support for reading data from the standard output stream and standard error stream of an application.
 
     An application cannot read data from the standard output stream and standard error stream because they are redirected. So the APIs that read characters from the stream, returns an empty or null string. For example, `Process.StandardOutput.ReadToEnd()`.
 
-    In case of Process.OutputDataReceived and Process.ErrorDataReceived, an event occurs but null is passed as DataReceivedEventArgs.Data. Even these event handlers are not called when the stream is closed.
+    In case of `Process.OutputDataReceived` and `Process.ErrorDataReceived`, an event occurs but null is passed as `DataReceivedEventArgs.Data`. Even these event handlers are not called when the stream is closed.
 
 
-2. Access to process components for other process resources is not allowed.
+2. No access to process components for other process resources.
 
     The APIs, which get other process components such as `Process.GetProcesses()` and `Process.GetProcessesByName()`, throw an exception because an application is not allowed to get information about other processes.
 
     In addition, `Process.GetProcessById()` with other process ID also throws an exception.
 
 
-3. Limited System.IO.MemoryMappedFile support.
+3. Limited support for System.IO.MemoryMappedFile.
 
-    Creating MemoryMappedFile with **execute** access capability is not supported. So `MemoryMappedFile.CreateViewAccessor()` and `MemoryMappedFile.CreateViewStream()` with both MemoryMappedFileAccess.ReadExecute and MemoryMappedFileAccess.ReadWriteExecute throw an exception.
+    Creating MemoryMappedFile with **execute** access capability is not supported. So `MemoryMappedFile.CreateViewAccessor()` and `MemoryMappedFile.CreateViewStream()` with both `MemoryMappedFileAccess.ReadExecute` and `MemoryMappedFileAccess.ReadWriteExecute` throw an exception.
```

Comment:
Creating MemoryMappedFile with **execute** access capability is not supported. 
In addition, `MemoryMappedFile.CreateNew()` and `MemoryMappedFile.OpenExisting()` are not supported because named maps are not supported.

From the above content, it shows that **No support for System.IO.MemoryMappedFile**.
Please correct me, if I am wrong.
If I am right, then, please changes "Limited" to "No" in heading **3**.
Please verify.

Reply:
There are other options in MemoryMappedFileAccess such as MemoryMappedFileAccess.CopyOnWrite and so on. So, "Limited" is right.

Reply:
OK. Then ignore my comments.
Thank you for explaining.